internal/core/application/validators.go

```diff
@@ -20,7 +20,7 @@ func validateAmount(satoshis decimal.Decimal) error {
 }
 
 func validateAssetString(asset string) error {
-	const regularExpression = `[0-9A-Fa-f]{64}`	
+	const regularExpression = `[0-9a-f]{64}`
```

Comment:
why changing the expression?

Reply:
asset hashes in hex string format should be all-lower-case we must not allow upper case ones.